contrib/guix/manifest.scm

```diff
@@ -123,6 +123,104 @@ desirable for building Bitcoin Core release binaries."
   (package-with-extra-patches mingw-w64-x86_64-winpthreads
     (search-our-patches "winpthreads-remap-guix-store.patch")))
 
+
+;; While mingw-w64 is packaged in Guix, we maintain our own package,
+;; to use UCRT runtime.
+(define* (make-mingw-w64/implementation machine
+                                             #:key
+                                             xgcc
+                                             xbinutils
+                                             with-winpthreads?)
+  "Return a mingw-w64 for targeting MACHINE.  If XGCC or XBINUTILS is specified,
+use that gcc or binutils when cross-compiling.  If WITH-WINPTHREADS? is
+specified, recurse and return a mingw-w64 with support for winpthreads."
+  (let* ((triplet (string-append machine "-" "w64-mingw32")))
+    (package
+      (name (string-append "mingw-w64" "-" machine
+                           (if with-winpthreads? "-winpthreads" "")))
+      (version "12.0.0")
```

Comment:
Any reason to not use the v13.0.0 headers: https://www.mingw-w64.org/changelog/?